apps/web/src/components/patient-documents/PatientDocumentUpload.tsx

```
@@ -0,0 +1,687 @@
+            isDragOver && 'border-primary bg-primary/10 scale-[1.02]',
+            disabled && 'opacity-50 cursor-not-allowed',
+            !disabled && canUpload && 'cursor-pointer',
+            !canUpload && 'border-gray-300 bg-gray-50 cursor-not-allowed',
+          )}
+          onDragOver={handleDragOver}
+          onDragEnter={handleDragEnter}
+          onDragLeave={handleDragLeave}
+          onDrop={handleDrop}
+          onClick={() => canUpload && fileInputRef.current?.click()}
+          role='button'
+          tabIndex={disabled || !canUpload ? -1 : 0}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)